e2e/bare-client-auth/pyproject.toml

```diff
@@ -3,7 +3,7 @@ requires = ["hatchling"]
 build-backend = "hatchling.build"
 
 [project]
-name = "e2e_test"
+name = "client_auth_e2e_test"
```

Comment:
Could you use `-` instead of `_` here?

![image](https://github.com/user-attachments/assets/9dafe5f4-dc8c-4429-a95d-fda311339210)

Reply:
Could you also update other app names as well?